Client/Frontend/Strings.swift

```diff
@@ -600,13 +600,8 @@ extension String {
                 public static let Title = MZLocalizedString(
                     key: "FirefoxHomepage.Pocket.Footer.Title.v115",
                     tableName: "Footer",
-                    value: "Powered by %@.",
-                    comment: "This is the title of the Pocket footer on Firefox Homepage. placeholder will be for App Name")
-                public static let Subtitle = MZLocalizedString(
-                    key: "FirefoxHomepage.Pocket.Footer.Subtitle.v115",
-                    tableName: "Footer",
-                    value: "Part of the %@ family.",
-                    comment: "This is the subtitle of the Pocket footer on Firefox Homepage. The placeholder is the app name.")
+                    value: "Powered by %@. Part of the %@ family.",
```

Comment:
you're gonna need to update the string ID to 116 and this needs to get retranslated.

Reply:
Can we even uplift that to 115? It's kind of late for translations 🤔 @dnarcese

Reply:
Ah interesting. It was just a formatting change. We can leave this for 116